packages/flutter/lib/src/material/app.dart

```diff
@@ -20,15 +20,16 @@ import 'package:flutter/foundation.dart';
 import 'package:flutter/services.dart';
 
 import 'arc.dart';
+import 'button_style.dart';
 import 'colors.dart';
-import 'floating_action_button.dart';
 import 'icon_button.dart';
 import 'icons.dart';
 import 'material_localizations.dart';
 import 'page.dart';
 import 'scaffold.dart' show ScaffoldMessenger, ScaffoldMessengerState;
 import 'scrollbar.dart';
 import 'theme.dart';
+import 'theme_data.dart';
```

Comment:
theme.dart exports theme_data.dart, I don't think this is necessary.
Nitty, can clean up in a separate change.